### DISCUSSION TITLE
How to get absolute energy level of electronic states in ABACUS?

### DISCUSSION THREAD

**Discussion by hongriTianqi** [deepmodeling/abacus-develop#694] [Ideas]
## Background
The absolute energy level of electronic states is very useful in electrochemistry. The atomic orbital basis should in principle give indications of absolute energy levels of electronic states. Could ABACUS give the absolute energy level of electronic states through atomic basis calculations?

## Describe the solution you'd like
Give more instructions or more explanation on this issue.

**Comment by mohanchen** [comment]
I don't understand. If one uses pseudopotential, there is no "absolute energy" exists, what do you mean "absolute energy levels"?

**Comment by hongriTianqi** [comment]
> I don't understand. If one uses pseudopotential, there is no "absolute energy" exists, what do you mean "absolute energy levels"?

In order to get absolute energy level of electrons, one should firstly define where is energy zero. Sometimes it is defined as all electrons and nuclei are apart from each other indefinitely, sometimes it is defined as vacuum energy. I don't know either.

**Comment by mohanchen** [comment]
We will leave this as a discussion topic.

**Reply by floatingCatty** [reply]
Is there any updates of this topics?
